### ISSUE TITLE
Training using MSE than finetuning using MS-SSIM but aux loss goes bigger

### ISSUE THREAD

**Issue by gitlabspy** [InterDigitalInc/CompressAI#232]
Following https://interdigitalinc.github.io/CompressAI/zoo.html#training, I first train with mse with $\lambda$ 0.18 and switch to ms-ssim with $\lambda$ 220, but aux loss goes bigger and bigger. Is this normal? Should I finetune the model with ms-ssim using lower learning rate like 1e-5?

**Comment by YodaEmbedding** [comment]
`aux_loss` being large on its own isn't a problem. Typical values I have observed are `aux_loss ~= 15 * batch_size`, but this varies depending on factors such as the number of channels, and the batch-to-batch stability of the distributions. But if you want to reduce it to 0, you can try https://github.com/InterDigitalInc/CompressAI/pull/231.
